diff --git a/SConstruct_ORIGINAL b/SConstruct
index 1ff2067..68295ab 100644
--- a/SConstruct_ORIGINAL
+++ b/SConstruct
@@ -1386,7 +1386,7 @@ if env['IMPORT_ENV']:
 
 # Check for unknown variables in case something is misspelled
 unknown = opts.UnknownVariables()
-if unknown:
+if unknown and not env['USE_UNKNOWN_VARS']:
     print "Unknown variables:", unknown.keys()
     print 'If you are sure these are right (e.g. you want to set some SCons parameters'
     print 'that are not in the list of TMV parameters given by scons -h)'
